.gitignore

```diff
@@ -68,3 +68,15 @@ vhdl/thunderbots_vhdl.prj
 vhdl/work-obj93.cf
 vhdl/xlnx_auto_0_xdb
 vhdl/xst
+
+# BEGIN setup.sh AUTOGENERATED
```

Comment:
Is this still autogenerated? If not we should update these comments.

Reply:
It is indeed autogenerated. There's an `ex` script and a function that automatically syncs the repository list from the array in `setup.sh` to the `.gitignore` on every run.